### ISSUE TITLE
Experimental Design LOA1 Identity Verification Alert

### ISSUE THREAD

**Issue by eileen-coforma** [department-of-veterans-affairs/vets-design-system-documentation#2284]
## What
There are multiple and inconsistent LOA1 alert styles and messaging used on VA.gov right now. We propose to update the LOA1 error alert to use the updated sign in or tool prompt alert style, and have consistent plain-language messaging that includes an action link within the alert itself.
<details><summary>Example of previous / other existing LOA1 error alert</summary>
<img width="779" alt="Screenshot 2023-11-27 at 12 55 53 PM" src="https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/111457818/e517f7fe-d245-4eea-b4bc-47a8a2111f62">

![Screenshot 2023-11-29 at 3 14 29 PM](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/111457818/510dff12-8cab-431c-b7d5-f16b549dcf60)

![Screenshot 2023-11-29 at 3 13 43 PM](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/111457818/bd789590-5b8b-4543-ab2c-83984c38e36a)

![Screenshot 2023-11-29 at 3 13 06 PM](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/111457818/6dcca6e3-4f37-4d7c-802b-d46c7a087285)
</details>

## Purpose
The alert prevents users from moving forward in an authenticated application and notifies them about the need to verify their account. It also provides an action link to verify their identity. The update provides message consistency across the platform.

## Usage
 **Context or task**: As a gate for authenticated applications, which require further account verification. It should appear on the introduction, preventing the user from starting the form.

## Behavior
It is an alert that shows within the page, with an action link, for users with LOA1 access that require verification to access application/tool. Action link will lead to https://www.va.gov/verify/

## Examples

Before an authenticated form (ie. Supplemental Claim 20-0995
![Screenshot 2023-11-29 at 12 58 16 PM](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/111457818/62056bbb-dafd-4d7f-abbd-d0fc6e472840)

## Accessibility
> Include any accessibility considerations.

## Guidance
- Use in cases where there is an authenticated app that requires identity verification before accessing. 
- Shown on the application page (for example MyVA where the Claim Status Tool is suppose to show) or before the application (for example form introduction).
- Plain-language message body and action link language should remain the same.
- Alert header could be tailored for more specific messaging. 
- DO NOT automatically redirect the user to the verification page.

## Research (optional)
> Include any research you have already conducted, or plan to conduct, on this component or pattern.

## Code (optional)
```
<va-alert status="continue">
  <h2 slot="headline">
    You'll need to verify your identity to access more VA.gov tools and
    features
  </h2>
  <p>
    We need to make sure you're you — and not someone pretending to be you —
    before we can give you access to your personal and health-related
    information. This helps to keep your information safe, and to prevent
    fraud and identity theft.
  </p>
  <strong>This one-time process takes about 5-10 minutes.</strong>
  <p>
    <a
      href={`/verify?next=${pathname}`}
      className="verify-link vads-c-action-link--green"
    >
      Verify your identity to start your request
    </a>
  </p>
</va-alert>
```
`pathname` is the URL pointing back to your app

## Next steps
You may present your work to the Design System Council at an upcoming meeting. If you do not or cannot attend the Design Council Meeting, you can opt to get an asynchronous approval. 

Submit requests to join an upcoming Design System Council meeting in #platform-design-system.  

During the meeting, the Design System Council Working Group will evaluate the request and make a decision. 

If your request is approved, you can [add your component or pattern to the system](https://design.va.gov/about/contributing-to-the-design-system/add-a-component-or-pattern-once-approved). If you have any questions on how to add your component or pattern to the system, please reach out to the Design System Team at #platform-design-system.

**Comment by asg5704** [comment]
👋 @eileen-coforma this is awesome to see. I work on the Identity team and would like some of our UX folks in here to give additional insights/guidance. @claytonzook

**Comment by eileen-coforma** [comment]
👋 @asg5704 That would be great! The council had mentioned we needed more insight from the ID team. This work is actually now being pursued by @rachelshearerux's team, since they have a form close to testing.

**Comment by asg5704** [comment]
Oh awesome thank you!

**Comment by laurwill** [comment]
Hi @rachelshearerux @asg5704 @eileen-coforma , I noted this work for @DanielleThierryUSDSVA 's awareness, since this type of updated, standardized content related to sign-in options would need to go through the new identity comms review process. We’re also working on updated language about sign-in options that we'll start to use in January, so we’ll plan to follow up about this in the new year. Let me know if you have any questions in the meantime!

(And we'll work separately with Rachel's team on the specific language in the form that's close to testing.)

**Comment by claytonzook** [comment]
I'm looking into the listed  5-10 minute estimate timing to go through the verification process. I noticed that neither LoginGov nor IDme have an estimated timing posted on their sites
- LoginGov has said time widely varies, and they might suggest about 10 minutes on average (see this [Slack exchange](https://dsva.slack.com/archives/C01CLV49YMB/p1702055453453379))
- I'm meeting with IDme later this week and this is among the agenda items to discuss

**Comment by humancompanion-usds** [comment]
@laurwill - Is this issue now superseded by CAIA's updates to the Sign in Alerts? https://github.com/department-of-veterans-affairs/va.gov-team/issues/76785

**Comment by laurwill** [comment]
Yes! The new sign-in alert variations will also provide standardized language & design for these "Verify" alerts.

**Comment by humancompanion-usds** [comment]
The new sign-in alerts are available in guidance: https://design.va.gov/components/alert/alert-sign-in/

The new web-component variations of Alert should be in Storybook end of next week and through staging review in the next sprint. We're trying to expedite given a number of teams looking to use these new variations.

**Comment by humancompanion-usds** [comment]
This work has shipped. An additional variation is being added for unauth forms.
